internal/provider/nordvpn/api/group.go

```diff
@@ -0,0 +1,20 @@
+package api
+
+type group struct {
+	Title string
+	Type  struct {
+		Identifier string
+	}
```

Comment:
Specify the json fields

```suggestion
	Title string `json:"title"`
	Type  struct {
		Identifier string `json:"identifier"`
	} `json:"type"`
```